drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -482,6 +646,19 @@ def self.sections # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexit
           title: 'Possible Missed Exit - ES, No Service in 90 Days or More',
           description: 'There is an expectation that clients will not stay indefinitely in emergency shelter, these clients have been in shelter more than 90 days',
```

Comment:
Should these say something like
```suggestion
          description: 'There is an expectation that clients will not stay indefinitely in emergency shelter, these clients have not received service in more than 90 days',
```
While the assertion is true, the comparison is against `days_since_last_service` not `lot`

Reply:
nope, those 3 are about not staying in shelter too long, "Possible Missed Exit - ES, No Service in 90 Days or More" etc. cover the not receiving service.